docker-compose.yaml

```diff
@@ -101,16 +101,29 @@ services:
       - PGDATABASE=dev
       - PGSSLMODE=disable
       - REST_PORT=3005
-      - USER_SERVICE_PORT=3000
     restart: on-failure
     depends_on:
       postgres:
         condition: service_healthy
-    healthcheck:
-      test: wget --spider --no-verbose --tries=1 localhost:3005/ || exit 1
-      interval: 5s
-      timeout: 5s
-      retries: 10
```

Comment:
Why remove the health check?

Reply:
reverted. However, sometimes the `matching-service` isn't started even though `collaboration-service` is healthy when performing ` docker compose -f .\docker-compose.dev.yaml up -d`. 
Do you encounter it?

Reply:
I've encountered this yeah. Not sure but a restart fixes it.